Library/Formula/node.rb

```diff
@@ -25,6 +25,7 @@ class Node < Formula
   option "without-completion", "npm bash completion will not be installed"
 
   depends_on :python => :build
+  depends_on :xcode => :build
```

Comment:
It's curious the ` npm ` team thought otherwise. Blurgh. I hate it when things break for a reason that's not completely obvious. Particularly, the fact it breaks on certain CLT-only machines, and resolves when we create a bottle that includes the ` npm make install ` stage, which would imply the lack of ` xcode ` is the issue, but if Node said otherwise, and ` npm ` are pretty sure there's nothing in ` npm ` that requires ` xcode ` then? Blurgh.